Makefile

```diff
@@ -19,6 +19,17 @@ build-with-podman:
 test:
 	go test -race ./... -v
 
+MOCKDIR = ./mocks
+.PHONY: mocks-gen
+mocks-gen: mocks-rm ## Generate mocks for all the defined interfaces.
+	go install github.com/golang/mock/mockgen@latest
```

Comment:
lets fix the mockgen version to the version of the mock package as recommended by the mock docs

Reply:
alright, done!